manifests/service.pp

```diff
@@ -120,8 +120,7 @@
   }
 
   $_manage_service = $manage_service ? {
-    false   => undef,
-    true    => 'Service[docker]',
+    true    => Service['docker'],
     default => undef,
```

Comment:
Yes, it was. I was just testing the newer versions of the module (I fixed
my git sha) and had the exact same issue as you with the quoted resources.
Only when I fixed that part, I thought to try with no resources (manage =>
false) and got the next problem. So I thought I would add this comment to
your PR :-)

Met vriendelijke groet,
Jo Vandeginste

jo.vandeginste@gmail.com

2016-02-29 16:15 GMT+01:00 Pi3r <notifications@github.com>:

> In manifests/service.pp
> <https://github.com/garethr/garethr-docker/pull/453#discussion_r54422225>:
>
> > @@ -120,8 +120,7 @@
> >    }
> >
> >    $_manage_service = $manage_service ? {
> > -    false   => undef,
> > -    true    => 'Service[docker]',
> > +    true    => Service['docker'],
> >      default => undef,
>
> @jovandeginste <https://github.com/jovandeginste> Thanks I have just
> forced a push with this change. Is it looking good ?
>
> I am actually a bit surprised as the PR did not change anything in that
> regard. I mean it was undef before.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/garethr/garethr-docker/pull/453/files#r54422225>.
>